climada/hazard/relative_cropyield.py

```diff
@@ -166,32 +167,33 @@ def set_from_single_run(self, input_dir=None, bbox=BBOX,
             LOGGER.warning('Input directory %s not set', input_dir)
             raise NameError
 
-
         yearchunk = YEARCHUNKS[scenario]
-        string = '%s_%s_ewembi_%s_%s_%s_yield-%s-%s_%s_%s_%s.nc'
-        filename = os.path.join(input_dir, string % (ag_model, cl_model, scenario, soc, co2, crop,
-                                                     irr, fn_str_var, str(yearchunk['startyear']),
-                                                     str(yearchunk['endyear'])))
+        filename = os.path.join(input_dir,
+                                '%s_%s_ewembi_%s_%s_%s_yield-%s-%s_%s_%s_%s.nc' \
+                                    %(ag_model, cl_model, scenario, soc, co2, crop,
+                                      irr, fn_str_var, str(yearchunk['startyear']),
+                                      str(yearchunk['endyear'])))
 
         # define indexes of the netcdf-bands to be extracted, and the
         # corresponding event names and dates
         # corrected indexes due to the bands in input starting with the index=1
         id_bands = np.arange(yearrange[0] - yearchunk['startyear'] + 1,
                              yearrange[1] - yearchunk['startyear'] + 2).tolist()
-        event_list = [str(n) for n in range(int(yearrange[0]), int(yearrange[1] + 1))]
 
         # hazard setup: set attributes
         self.set_raster([filename], band=id_bands,
                         geometry=list([shapely.geometry.box(bbox[0], bbox[1], bbox[2], bbox[3])]))
```

Comment:
An easier to read way to do:
`[Lon min, lat min, lon max, lat max] = bbox
  self.set_raster([filename], band=id_bands,
         geometry=list([shapely.geometry.box(Lon min, lat min, lon max, lat max)]))`